### PR TITLE
Project of Teardown Design For Microverse

### PR DESCRIPTION
**Pull request by dasileker**
**Project of Teardown Design For Microverse**

> Request For a code Review
_In This Project We worked With:_

- HTML & CSS 

- Images & SVG 

- README.MD File Contain all the information about the  project

### CODE REVIEW

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested

Hello @dasileker and @TSHEPO-CLOUD ! 👋

Awesome job so far!! 👍 you still need to make some changes though!

## Mandatory Changes ♻️

- [x] your project is missing one of the linter files `.stylelintrc.json`, please add it and recheck for errors 
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))

- [x] remove the file named "image" as it's an empty file that you don't need

### Navigation bar

* [x] It uses `nav` element with `ul`/`li` inside, however, students are allowed to used `div` as the main focus is heat map. (content is not important) ([link to page](https://www.smashingmagazine.com/))

### Each article

* [x] It uses [`article` tag](https://www.w3schools.com/tags/tag_article.asp) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/design_teardown/article_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540025/))

### Optional

* [x] use of the [`<nav>` tag](http://html5doctor.com/nav-element/) for the major navigation links
* [x] use of the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document
* [x] use of the [`<footer>`](https://www.w3schools.com/TagS/tag_footer.asp) tag for the footer
* [x] don't use a Class and an ID with the same name, as it might confuse whoever reading your code! instead, use names that describe your section
* [x] make your branch name descriptive, maga is not a good name!


### please make the appropriate changes and ask for another code review ✔️
Happy Coding!

Best,
Aziz Mejri
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@Spayco) if you need any help_

**Comment by dasileker** [comment]
hello, and thank you for the review I think you're wrong the .stylelintrc.json  is there but I add it to the gitigonre file and second, I did check the web hint  there is no problem using  any tag name for semantic styling

**Comment by SpaYco** [comment]
> hello, and thank you for the review I think you're wrong the .stylelintrc.json is there but I add it to the gitigonre file and second, I did check the web hint there is no problem using any tag name for semantic styling

you should not add it to .gitignore, as GitHub needs it to scan the files! and i wouldn't have any idea that you actually used it to scan :)

**Comment by dasileker** [comment]
You see this green thing that says all checks passed he is the linter file you said 

 You can click on the right top link shiw all check

**Comment by SpaYco** [comment]
the checks can pass even without the file, please include it in the files and remove it from .gitignore.

also, delete the "image" file as it's an empty file

**Comment by dasileker** [comment]
Man you're not a TSE and  you're totally wrong all my previous projects where the same as this one, and they all pass, i don't want to  debate  but thanks i will wait for a real TSE to review my code

**Comment by SpaYco** [comment]
again, the `.stylelintrc.json` should be on the repo, not on your local folder, in order for GitHub Actions to do the testing, 

maybe the previous TSEs only saw the checks and didn't check the files, but the file should be there. 

this project will not be approved unless you do so, as this is a **mandatory step**

**Comment by dasileker** [comment]
Okay and thanks

**Comment by SpaYco** [comment]
Hey again, just talked to the head of TSE, turned out that GHA uses the files through links even if files didn't exist on the repo. 

you can continue without adding the file, but fix the other issues.

**Review by jofer86 — CHANGES_REQUESTED** [review]
## :wave: :wave: Hello! I'm Jorge, and  I will be conducting this code review. Let me get a closer look 🔎 🔍 🔎 🔍 🕵🏼 
- Changes marked as **[OPTIONAL]** are not hard requirements but are nice to have.
- First of I would like to urge to act professionally in the interactions with the TSE's, and with your partners in general 
image
That is not the way to professionally approach a colleague, fellow student or TSE.  Respect and cordiality are important values a software developer must have in order to be successful so please refrain from discussing things in a rude matter, it's just not nice.
- Back to the project, it seems that you missed the core of the project, which is providing a heat map of the visual hierarchy of the site currently it has only two shades of gray when you should be considering more shades as there are not only two visual hierarchy present in the project.  The higher a component is on the hierarchy (the more your sight goes to that component)  the darker it should be.
- The first section of the project is not lined up correctly the website shows this 
image
- While yours shows this 
image
- It should be a three article row instead of a 2x2.
-  **[OPTIONAL]** There is a big space on the right side of the screen that is not being considered.
-  **[OPTIONAL]** You should change your font to a sans-serif that's more web-friendly.
- Keep your interactions respectful and polite, please.
## ` const STATUS = 'Changes Required';` 😿 😿 😿
--------------------------------------------------------------------
**Rock on**  🤘🏼 
**_Jorge Fernando Rincón Figueroa._** 
Technical Support Engineer.

**Comment by dasileker** [comment]
### HELLO @jofer86 
 
- maybe I'm not professional  but also people who are not  knowing what they do  and they think I'm a stupid 

1. I'm an Udacity full-stack web dev graduated and this work just to give the idea I'm a beginner.

2. the page in your browser is  75% zoom so the original will show 3 columns, not 2-column

3. please adjust the zoom % of your browser to see 

this is the link so you can see https://www.smashingmagazine.com/

and thank you for the review it helps me to see  the difference

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
# Status: Changes requested :red_circle: 

## Hello @dasileker and @TSHEPO-CLOUD :wave: 

## Your page is looking good, I can see you put a lot of effort into it :muscle: 

### Thank you for making the required changes above, but we still need to fix your background colors. As @jofer86 said, this is a heatmap, you need to show the hierarchy through colors and fonts. Please check your navbar, it has the same shade of grey as the article on the left, while the original website managed to show a different shade of red:

## Yours:
Screenshot from 2020-08-14 18-26-23

## Original:
Screenshot from 2020-08-14 18-26-19

<br>

## Kindly make the requested changes and submit for another review, please :pray: 

## I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Comment by dasileker** [comment]
### thank you @MuriloRoque  
**changed are made I hope it's good**

File: index.css
Comment (expjazz) [inline index.css:6]: - [x] Kindly change the background color of the navbar to a different kind of gray so it is different than the article right below (like the previously TSE asked).

| Yours | Original |
|:---:|:---:|
|image |image|

image

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle: 

#### Hello Team. I'm Expedito and I will review this PR.

Your project looks awesome. The project is almost completed :clap: However, the last TSE asked you to change the background color of the navbar to be different from the left article of the section 'Dont Miss these latest Articles'. (Check the inline comments below). Change the background color of the navbar to a different tone of gray.

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:

File: index.css
Comment (dasileker) [reply index.css:6]: ok done

**Review by MuriloRoque — APPROVED** [review]
# Status: Approved :green_circle: 

## Hello again @dasileker :wave:

## Congratulations, your project is approved, your page really looks great! :grin: 

## You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**
